### ISSUE TITLE
Header and columns don't align when at > 100% zoom

### ISSUE THREAD

**Issue by jmcphers** [DataTables/Scroller#44]
Repro (this in Chrome 40.0.2214.115 on OS X 10.10.2, but seems to repro elsewhere):

1. Go to https://datatables.net/extensions/scroller/
2. Increase zoom level 
3. Reload page
4. Look closely at the headers--they don't align with the content

![image](https://cloud.githubusercontent.com/assets/470418/6399499/ac9cb858-bda7-11e4-83e7-f1fcc0521d06.png)

Seems like a spurious `padding-left: 14px` is being added?

**Comment by DataTables** [comment]
Chrome 40 on the exact same revision and OS and I don't seem to be able to reproduce this.

Zoom level in browsers is just a nightmare to support. It is different in every browser and they are all buggy... Generally I don't support anything other than 100% zoom - although obviously it is a nice to have if it works correctly! Any other way to reproduce this, or perhaps you could try inspecting the element to see if it does have padding?

**Comment by jmcphers** [comment]
Yes, it has padding, and it's even explicitly assigned!

    <div class="dataTables_scrollHeadInner" style="box-sizing: content-box; width: 869px; padding-left: 14px;">

![image](https://cloud.githubusercontent.com/assets/470418/6400025/2635ba9a-bdab-11e4-88e3-8a3e0a7bfda4.png)

Did you reload the page? This only repros for me if the page's first render is at > 100% zoom -- just zooming in on an already rendered DataTable is fine.

**Comment by jmcphers** [comment]
Also repros on Safari 8, but does not seem to repro on Firefox 33 (maybe WebKit related?).

**Comment by DataTables** [comment]
Bizarrely I can't reproduce on Safari either.

Exact steps I've taken:

1. Load page
2. Zoom using cmd-+
3. Reload
4. All the same.

Ooo. I think I might know what it is - the left-to-right detection in DataTables... I remember getting a report a little while back that when the zoom level was adjusted it made DataTables think it was right-to-left, so it puts space for the scrollbar on the left...

I'll bet that is what it is.

Could you do me a favour and on the page when that problem is showing open the console and punch in:

    $.fn.dataTableSettings[0].oBrowser.bScrollbarLeft

My guess is that it will show `true`...

**Comment by jmcphers** [comment]
Yes, that's it! 

    > $.fn.dataTableSettings[0].oBrowser.bScrollbarLeft
    true

**Comment by DataTables** [comment]
Got it - thanks!

It wasn't appearing on my machine as I've got (had) this machine configured to Apple's new scrollbars - zero width and floating. With scrollbars shown it was immediately visible.

The problem it turns out was indeed the left scrollbar detection - [when DataTables checks for the position](https://github.com/DataTables/DataTablesSrc/blob/master/js/core/core.compat.js#L230) of an inner element that it set to be 1px, it expected to get back 1px, but when zoomed it was actually returning a non-integer value: 0.9943181276321411.

I've added a call to `Math.round` which should address the problem.

Thanks for your help investigating this.

[This bug on the DataTables core repo](https://github.com/DataTables/DataTables/issues/479) also notes this issue.
